src/wp-includes/functions.php

```diff
@@ -2087,6 +2098,16 @@ function wp_mkdir_p( $target ) {
 			}
 		}
 
+		/**
+		 * Fires after the directory is created and the permissions are set.
+		 *
+		 * @since 6.4.0
+		 *
+		 * @param string $target Full path to the created directory.
+		 * @param int $dir_perms Directory permissions
```

Comment:
```suggestion
		 * @param string $target       Full path to the created directory.
		 * @param int      $dir_perms Directory permissions.
```
- The types and descriptions used in `@param` annotations should be aligned.
- Parameter descriptions should end with a period.